### PR TITLE
Enable utility feature

### PR DESCRIPTION
**Pull request by Fabi0-Fl0res**
This is the initial implementation of the utility feature based on the requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (Fabi0-Fl0res) [inline util/config.txt:3]: This config value should be 567. 
The old value was 123.

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
+
+feature 3 implementation
```
Comment (Fabi0-Fl0res) [inline util/feature.txt:5]: Let's add a bit more to the feature 2 implementation.
I think it should handle multiple widgits.

File: util/config.txt (hunk 1 above)
Comment (fabio248) [inline util/config.txt:3]: This config value should be 567. 123 was the old value.
The old value was 123

File: util/feature.txt (hunk 2 above)
Comment (fabio248) [inline util/feature.txt:5]: Let's add a bit more to the feature 2 implementation.
I think it should handle multiple widgits.

Comment (Fabi0-Fl0res) [reply util/feature.txt:6]: @fabio248 see the latest commit for the change

**Review by Fabi0-Fl0res — COMMENTED** [review]
Modified code based on review

**Review by fabio248 — APPROVED** [review]

**Pull request merged**